wp-content/themes/core/styles/blocks/heading/b-x-small.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Rather than naming these with letters, can we use a 10-based index system? Ie.  `10-xx-small.json`, `20-x-small.json` etc? using 10-based indexes allows for adding new variations between existing ones easier.